[release_note_tool] Update test and skip commits that errors out

jerryzh168 opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* __->__ #108252

Summary:
att

Test Plan:
python test_release_notes.py

Reviewers:

Subscribers:

Tasks:

Tags:

jerryzh168 committed:
[release_note_tool] Update test and skip commits that errors out

Summary:
att

Test Plan:
python test_release_notes.py

Reviewers:

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

drisspg commented on scripts/release_notes/common.py:
```diff
@@ -253,7 +253,7 @@ def github_data(pr_number):
     )
     query = run_query(query)
     if query.get("errors"):
-        raise Exception(query["errors"])
```
I think we want to be loud about the number of errors that occur. Since if 1 or 2 commits are busted that is probably fine but could this potential hide some problems?

jerryzh168 replied:
I see, makes sense, I can add some handling for that

jerryzh168 committed:
Update on "[release_note_tool] Update test and skip commits that errors out"


Summary:
att

Test Plan:
python test_release_notes.py

Reviewers:

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

jerryzh168 committed:
Update on "[release_note_tool] Update test and skip commits that errors out"


Summary:
att

Test Plan:
python test_release_notes.py

Reviewers:

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

jerryzh168 commented:
@pytorchbot merge
